This patch is needed to get xine-lib build with SDL support in FreeBSD. Ports
by default installs the header as <SDL11/SDL.h>, while upstream is just <SDL.h>
this patch changes the check from a generic __FreeBSD__ check to an actual
autoconf check so it works on both Gentoo/FreeBSD and ports.

Index: xine-lib-1.1.1/src/video_out/video_out_sdl.c
===================================================================
--- xine-lib-1.1.1.orig/src/video_out/video_out_sdl.c
+++ xine-lib-1.1.1/src/video_out/video_out_sdl.c
@@ -47,7 +47,8 @@
 #include <string.h>
 #include <inttypes.h>
 #include <math.h>
-#if defined (__FreeBSD__)
+
+#ifdef HAVE_SDL11_SDL_H
 # include <SDL11/SDL.h>
 #else
 # include <SDL.h>
Index: xine-lib-1.1.1/configure.ac
===================================================================
--- xine-lib-1.1.1.orig/configure.ac
+++ xine-lib-1.1.1/configure.ac
@@ -915,7 +915,10 @@ AC_ARG_ENABLE([sdl],
 
 if test "x$with_sdl" = "xyes"; then
   AM_PATH_SDL(1.1.5, 
-    AC_DEFINE(HAVE_SDL,1,[Define this if you have SDL library installed]),
+    [
+     AC_DEFINE(HAVE_SDL,1,[Define this if you have SDL library installed])
+     AC_CHECK_HEADERS([SDL11/SDL.h])
+    ],
     [])
 else
   no_sdl=yes
